packages/venia-concept/src/components/CategoryList/categoryList.js

```diff
@@ -66,30 +77,51 @@ class CategoryList extends Component {
 
                         return (
                             <div className={classes.content}>
-                                {data.category.children.map((item, index) => (
-                                    <Link
-                                        className={classes.item}
-                                        to={`/${
-                                            item.url_key
-                                        }${categoryUrlSuffix}`}
-                                        key={index}
-                                    >
-                                        <span className={classes.imageWrapper}>
-                                            {item.image && (
-                                                <img
-                                                    className={classes.image}
-                                                    src={`/pub/media/catalog/category/${
-                                                        item.image
-                                                    }`}
-                                                    alt={item.name}
-                                                />
-                                            )}
-                                        </span>
-                                        <span className={classes.name}>
-                                            {item.name}
-                                        </span>
-                                    </Link>
-                                ))}
+                                {data.category.children.map((item, index) => {
+                                    let imagePath;
```

Comment:
I believe it will be better to move logic of making imagePath to separate helper function. In general, I propose to create file helpers.js in component folder and move some local helper functions, which is used only in this component there. In this case we can call helper `getImagePath`;